Add instructions to set up ld alias (#189)
Exove/local-docker: README.md

connect to on your host. Port `9010` is being used to avoid collision
with possible running `php-fpm` on the host machine port `9000`.

#### Project level alias for ./ld.sh

You can set up a "local-docker finder" via Bash/Zsh function, which will try to
find the project's ld.sh script, and pass command name and other argumets to it.

Add this to your `~/.zshrc` or `~/.bashrc`:

    # Local-docker tool
    #

>     # This shell function travers upwards in the folder tree starting form the

Reviewer: ```suggestion
    # This shell function travels upwards in the directory tree starting from the
```

- Either 'traverses' or 'travels'.
- "folder" or "directory"? The latter IMHO but YMMV.
- form -> from

>     # current folder and up to the user's home folder (or filesystem root).

Reviewer: ```suggestion
    # current directory and up to the user's home directory (or filesystem root).
```
helping hand in case folder < directory

>     # If ld.sh file is found and is executable shell script, execute it passing

Reviewer: ```suggestion
    # If the ld.sh file is found and is an executable shell script, execute it passing
```

    # all provided arguments forward.
    #
    function ld() {
      CURRENTLY_CHECKED=`pwd`
      FOUND=

      while [ "${#CURRENTLY_CHECKED}" -gt "1"  ] && [ –z "$FOUND"] ; do

        [ -x "${CURRENTLY_CHECKED}/ld.sh" ] && FOUND="1"

        if [ -n "$FOUND" ]; then

>           ${CURRENTLY_CHECKED}/ld.sh "$@"

Reviewer: I am a bit worried about this bit based on the discussion in our company Slack on Apr 1 with @hirvinen. As I'm not too well versed with bash and arguments passing, this might be alright as well.

Author: I think it earns a whole refactoring approach, there probably are many places where a better approach would be more suitable than the current one. Shall we leave it to that refactoring round?

Reviewer: Let's.

>           # Pass the recevied return code as-is.

Reviewer: ```suggestion
          # Pass the recieved return code as-is.
```

Author: recieved vs received?

          return $?
        else

>           # Shorten the folder path by one directory.

Reviewer: ```suggestion
          # Shorten the directory path by one.
```

          NEXT_UP=`dirname $CURRENTLY_CHECKED`

>           #If we reached the user's home folder stop.

Reviewer: ```suggestion
          #If we reached the user's home directory, stop.
```

          if [ "$CURRENTLY_CHECKED" = "$HOME" ]; then

>             echo "ld.sh not found in this or any of the parent folders. Traversed up the tree until reached home folder ($HOME)."

Reviewer: ```suggestion
            echo "ld.sh not found in this or any of the parent directories. Traversed up the tree until reached home directory ($HOME)."
```

            return 1
          elif [ "${#NEXT_UP}" -eq "1"  ]; then

>             echo "ld.sh not found in this or any of the parent folders. Traversed up the tree until reached the filesystem root."

Reviewer: ```suggestion
            echo "ld.sh not found in this or any of the parent directories. Traversed up the tree until reached the filesystem root."
```

            return 1
          fi
          CURRENTLY_CHECKED=$NEXT_UP
        fi

      done
    }

Source the file (or start a new shell window) and you can use `ld` to execute
your local-docker commands.

> In order to not interfere with `/usr/bin/ld`, you may use another name besides

Author: will do

> `ld` as function name. However, it is quite rarely used command.

Reviewer: ```suggestion
`ld` as function name. However, it is quite rarely used a command.
```
